### PR TITLE
Add project name to populate analysis

### PR DESCRIPTION
**Pull request by illusional**

### CODE REVIEW

File: scripts/populate_sm_analysis.py [hunk 1]
```diff
@@ -116,9 +141,11 @@ def _populate_qc_analysis_entries(multiqc_json_path: Path):
             analysis_type='joint-calling',
             analysis_status='completed',
             target=cohort,
-            meta=cohort.get_job_attrs() | dict(
+            meta=cohort.get_job_attrs()
+            | dict(
                 sequencing_type=cohort.sequencing_type.value,
             ),
+            project_name='seqr',
```
Comment (vivbak) [inline scripts/populate_sm_analysis.py:148]: Is this a remnant from testing?

Comment (illusional) [reply scripts/populate_sm_analysis.py:148]: Intentionally left here, joint-calling results should go into the `seqr` project (we joint-call across datasets)

File: cpg_pipes/providers/cpg/status.py [hunk 2]
```diff
@@ -88,7 +88,8 @@ def create_analysis(
         analysis_type: str,
         analysis_status: str,
         target: Target,
-        meta: dict | None = None,        
+        meta: dict | None = None,
+        project_name: str=None,
```
Comment (vivbak) [inline cpg_pipes/providers/cpg/status.py:92]: I think the linter is complaining here 
```suggestion
        project_name: str = None,
```

File: scripts/populate_sm_analysis.py (hunk 1 above)
Comment (vivbak) [reply scripts/populate_sm_analysis.py:148]: ah, that makes sense :)

File: cpg_pipes/providers/cpg/status.py (hunk 2 above)
Comment (illusional) [reply cpg_pipes/providers/cpg/status.py]: Addressed

**Review by vivbak — APPROVED** [review]

**Pull request merged**